tasks/main.yml

```diff
@@ -40,3 +40,15 @@
 - name: Start kubelet
   become: yes
   service: name=kubelet.service state=started enabled=yes
+
+- name: make fstab directory for cifs plugin
+  become: yes
+  file:
+    path: "{{kubelet_volume_plugin_dir}}/fstab~cifs"
+    state: directory
+  when: kubelet_enable_cifs_mount
```

Comment:
I think you need to apply the Ansible `bool` filter to this check else `False` or `True` is treated as string making this always true.
Sorry for not noticing this earlier.

Reply:
Puneet, If we use False for this variable, it will skip installing the plugin.